.prettierrc

```
@@ -0,0 +1,5 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "singleQuote": false
+}
```

nit: add a new line at the end of the file

```suggestion
{
  "trailingComma": "es5",
  "tabWidth": 2,
  "singleQuote": false
}

```